innersource-activity-score.md

```diff
@@ -0,0 +1,92 @@
+## Title 
+InnerSource Activity Score
+
+## Patlet
+The InnerSource activity score is a numeric value that represents the (GitHub) activity of an InnerSource project. It can be used to create a ranked list of projects for the [InnerSource portal](innersource-portal.md) so that potential contributors can easily find the most active projects. It is derived automatically from GitHub repository statistics and may be enriched with KPIs from other tools or manual evaluations. 
+
+## Problem
+
+Potential contributors want to find active projects with a lot of traction, but also fairly new and enthusiastic projects with a need for new contributions - easy and fast.
+
+**But in which order** shall InnerSource projects be presented? Typical ranking KPIs like *GitHub Stars*, *Number of Forks*, *Number of Commits*, *Lines of Code*,  *Last Update* aren't very accurate when it comes to activity of a project.
+
+A new metric derived from several KPIs is needed to define a reliable and versatile score for a project's activity level.
+It can be used to sort or rank projects according to their activity level.
+
+## Story
+
+When InnerSource is practiced for a long time or scales beyond a certain number of projects (let's say 50 to give a meaningful threshold) it is hard to find the currently most popular and active InnerSource projects. Projects that exist for a long are well-known but may no longer be very active. Fairly new projects on the other hand don't have a repuation or an active community yet.
+
+A list of InnerSource projects should not be considered a static resource, but an exciting place to discover and explore new and active projects, just like a news page listing the most interesting topics of the day first. Thus it is beneficial when the order of the projects is regularly updated and changes according to the project's popularity and activity.   
+
+These considerations let to a first prototype to calculate an InnerSource activity score, which worked surprisingly well and determines an ever-changing order of projects according to their activity.
+
+## Context
+
+Discovering InnerSource projects can be facilitated with the [InnerSource Portal](innersource-portal.md) and the [Gig Marketplace](gig-marketplace.md) pattern, or by promoting projects on other communication channels and platforms. The activity score defines a default order or ranking in which projects are presented to the community. 
+
+## Forces
+
+Automated KPIs that can be fetched by querying the GitHub API are only part of the truth. What about code quality, the availability of good documentation, or an active and helping community that makes the project a fun place to contribute.
```

Comment:
```suggestion
Automated KPIs that can be fetched by querying the GitHub API are only part of the truth. What about code quality, the availability of good documentation, or an active and helping community that makes the project a fun place to contribute?
```

Reply:
Forces typically act upon the problem space in a way that is expressed through trade-offs. E.g., automated KPIs are easy to obtain but might not convey the whole picture. Soft KPIs are harder to gather but can add valuable meaning.

The "For now, we focus on a fully automated solution." isn't really a part of Forces but something that would appear in the Solution section (a decision on what to do with this particular Force).

Reply:
moved parts of it to solution, thx